src/hotspot/os/bsd/os_perf_bsd.cpp

```diff
@@ -220,71 +217,278 @@ int CPUPerformanceInterface::CPUPerformance::cpu_loads_process(double* pjvmUserL
   _jvm_system_nanos = jvm_system_nanos;
 
   return result;
+}
+
+#else // !APPLE
+
+CPUPerformanceInterface::CPUPerformance::CPUPerformance() {
```

Comment:
I notice that _jvm_ticks isn't initialised there.  I believe that means the initial values in it are undefined.  I don't see any problems that jump out at me, but I'd suggest doing an initialisation here anywhere to avoid incorrect assumptions in the future.